examples/nodejs/BUILD

```
@@ -4,23 +4,30 @@
 package(default_visibility = ["//visibility:public"])
 
 load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+load("//base:distro.bzl", "DISTRO_SUFFIXES")
 
 # These examples are adapted from:
 # https://howtonode.org/hello-node
-container_image(
-    name = "hello",
-    base = "//experimental/nodejs:nodejs10",
-    cmd = ["hello.js"],
-    files = [":hello.js"],
-)
+[
+    container_image(
+        name = "hello",
+        base = "//experimental/nodejs:nodejs10" + distro_suffix,
```

The build failed because this results in two `container_image` rules with the same name `hello`. Maybe
```suggestion
        name = "hello_" + distro_suffix,
        base = "//experimental/nodejs:nodejs14_" + distro_suffix,
```
(The code is untested.)